docs/decisions/0023-planner-telemetry-enhancement.md

```diff
@@ -0,0 +1,218 @@
+---
+status: { proposed }
+contact: { TaoChenOSU }
+date: { 2023-11-21 }
+deciders: alliscode, dmytrostruk, markwallace, SergeyMenshykh, stephentoub
+consulted: {}
+informed: {}
+---
+
+# Planner Telemetry Enhancement
+
+## Context and Problem Statement
+
+It would be extremely beneficial for applications using Semantic Kernel' planning features to be able to continuously monitor the performance of planners and plans.
```

Comment:
```suggestion
It would be extremely beneficial for applications using Semantic Kernel's planning features to be able to continuously monitor the performance of planners and plans.
```